VerilogCodingStyle.md

```diff
@@ -1527,6 +1526,33 @@ Do not use positional arguments to connect signals to ports.
 
 Instantiate ports in the same order as they are defined in the module.
 
+Align expressions in [tabular style](#tabular-alignment).
+
+***Use named parameters for all instantiations.***
+
+When parameterizing an instance, specify the parameter using the named parameter
+style. An exception is if there is only one parameter that is obvious such as
+register width, then the instantiation can be implicit.
+
+Indentation for module instantiation follows the standard indentation
+rule of two space indentation.
+
+```systemverilog
+my_module #(
+  .Height(5),
+  .Width(10)
+) my_module (
+  ...etc...
+
+my_reg #(16) my_reg0 (.clk_i, .rst_ni, .d_i(data_in), .q_o(data_out));
```

Comment:
Verible will format this to have each port connection on a new line (which I actually like, since it makes to port connections easier to read).